test/common/stats/tag_extractor_impl_test.cc

```diff
@@ -516,6 +516,16 @@ TEST(TagExtractorTest, DefaultTagExtractors) {
                          {proxy_protocol_version});
   regex_tester.testRegex("proxy_proto.test_stat_prefix.versions.v2.error", "proxy_proto.error",
                          {proxy_protocol_prefix, proxy_protocol_version});
+
+  // TLS certificates
+  Tag certificate_name;
+  certificate_name.name_ = tag_names.TLS_CERTIFICATE;
+  certificate_name.value_ = "server_cert";
+
+  listener_address.value_ = "0.0.0.0_0";
+  regex_tester.testRegex(
```

Comment:
Please also add a test case for a cluster (as the regex should do both listener and cluster)

Reply:
Added